frontend/src/features/admin-form/responses/queries.ts

```diff
@@ -90,10 +90,6 @@ export const useFormResponses = ({
     adminFormResponsesKeys.metadata(formId, params),
     () => getFormSubmissionsMetadata(formId, params),
     {
-      refetchOnMount: true,
-      refetchOnWindowFocus: false,
-      // Data will never change.
-      staleTime: Infinity,
       keepPreviousData: !submissionId,
```

Comment:
ensure the correct behaviour happens yea

Reply:
yes sir 🫡